README.rdoc

```diff
@@ -158,6 +158,13 @@ or
 
   minitest_bisect --seed 3911 test/**/*_test.rb
 
+==== Q: What if I need to run this on a typical Rails application?
+
+A: You would need to add the test/ directory in the search path like this: 
+
+   minitest_bisect --seed=3911 -Itest $(find test -type f -name \*_test.rb)
+
+
```

Comment:
Ah. OK. I see. I think this is better served as a single line change rather than adding a whole new section that doesn't really seem different. Can you just add `-Itest` to the previous Q&A?